### PR TITLE
Internationalization

### LINKED ISSUE #153: Make the site bilingual
Related: #16 

- [ ] Make a language preference flag thingy
- [ ] Render each page in both english and dutch, based on the texts provided by contentful

### PR DESCRIPTION
**Pull request by Mstiekema**
Updating the website so it is multilingual.

Fixes #153

### CODE REVIEW

**Comment by Mstiekema** [comment]
Next up is to update all pages with translations. They should read the storage, check the selected language and get that locale. I will look into a general query function for this, but I'm not sure if it's possible to implement it.

There are also a lot of untranslated lines in the application, so those have to be translated and implemented as well.

**Comment by Mstiekema** [comment]
This PR is almost done. I just have to fix some static pages (eg. membership, documents -> bugged since they don't have a page in src/pages) and some translations are missing. Other than that, it's pretty much done.

**Comment by Mstiekema** [comment]
I'm using both `localstorage.getItem()` and `location.href.split()` throughout the changes. Should this be changed to solely use the location or not? :thinking:

File: gatsby-config.js [hunk 1]
```diff
@@ -3,16 +3,18 @@ require('dotenv').config();
...
   plugins: [
     'gatsby-plugin-react-helmet',
     'gatsby-plugin-styled-components',
-    'gatsby-plugin-react-helmet',
     {
       resolve: 'gatsby-source-contentful',
       options: {
         spaceId: process.env.CONTENTFUL_SPACE_ID,
         accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+        environment: 'translations',
```
Comment (Mstiekema) [inline gatsby-config.js]: This environment is for getting the translation environment in contentful. When you remove this line, you will get the default contentful data.

File: gatsby-node.js [hunk 2]
```diff
@@ -86,26 +107,79 @@ exports.createPages = async ({ graphql, actions }) => {
...
+        createLanguageRedirect(node, '');
+      });
+    });
+  }
+
   if (query.errors) {
     throw new Error(JSON.stringify(query.errors));
   } else {
+    // Create all static pages
+    createStaticPages(staticFolder, '', createTemplatePage);
```
Comment (Riscky) [inline gatsby-node.js]: This seems to be one argument too much

**Comment by Phavion** [comment]
Awesome fix!

**Pull request merged**